flink-kubernetes/src/test/java/org/apache/flink/kubernetes/KubernetesClientTestBase.java

```diff
@@ -143,6 +147,34 @@ protected void mockGetDeploymentWithError() {
         server.expect().get().withPath(path).andReturn(500, "Expected error").always();
     }
 
+    protected void mockPodEventWithLabels(Map<String, String> labels) {
+        final Pod pod1 =
+                new PodBuilder()
+                        .withNewMetadata()
+                        .withNamespace("test")
+                        .withName("tm_pod1")
+                        .withLabels(labels)
+                        .withResourceVersion("5668")
+                        .endMetadata()
+                        .build();
+        // mock four kinds of events.
+        server.expect()
+                .withPath(
+                        "/api/v1/namespaces/test/pods?labelSelector=label1%3Dvalue1%2Clabel2%3Dvalue2&resourceVersion=0&allowWatchBookmarks=true&watch=true")
+                .andUpgradeToWebSocket()
+                .open()
+                .waitFor(1000)
```

Comment:
I'm not very familiar with this api. Can anyone tell me if this means we will definitely wait for `1` second or at most `1` second. If it is the former, it should definitely be prohibited because it will seriously slow down the execution time of `AZP`.

Reply:
definitely wait for 1 second